install.sh

```diff
@@ -62,6 +62,32 @@ Install()
     echo ""
     echo "4- ${installing}"
 
+    if [ "X$INSTYPE" != "Xagent" ]; then
```

Comment:
I think this code should belong to section 3 (installation form), maybe inside `ConfigureServer()` function.

Please move this segment to a function like `ConfigureBoot()` and call it just before `SetupLogs "3.8"` and also before `SetupLogs "3.6"`.

At the same time, increase `SetupLogs "3.8"` to `SetupLogs "3.9"` and `SetupLogs "3.6"` to `SetupLogs "3.7"`.

Thanks!